### PR TITLE
Add refund functionality

### PR DESCRIPTION
**Pull request by Piidgey**
PGW-532

### CODE REVIEW

File: altapay/transaction.py [hunk 1]
```diff
@@ -57,6 +57,28 @@ def capture(self, **kwargs):
 
         return altapay.callback.Callback.from_xml_callback(response)
 
+    def refund(self, **kwargs):
+        """
+        Refund full or partial payments on a order.
```
Comment (HenrikOssipoff) [inline altapay/transaction.py]: a -> an

File: docs/guide/transactions.rst [hunk 2]
```diff
@@ -137,3 +137,23 @@ As always, see the full list of possible arguments in the AltaPay documentation.
...
+See the AltaPay documentation for ``API/refundCapturedReservation`` for full details.
+
+.. code :: python
+
+    from altapay import Transaction
+
+    transaction = Transaction.find('TransactionID', api=api)
+    callback = transaction.refund()
+
+    if callback.result == 'Success':
```
Comment (estj92) [inline docs/guide/transactions.rst]: Consider doing this like "Releasing a Reservation" above.

File: docs/guide/transactions.rst [hunk 3]
```diff
@@ -137,3 +137,39 @@ As always, see the full list of possible arguments in the AltaPay documentation.
...
+
+            transaction = Transaction.find('TransactionID', api=api)
+            respnse = transaction.capture(
+                orderLines=[{
+                    'description': 'Blue Shirt',
+                    'itemId': '12345',
+                    'quantity': 1.0,
+                    'unitPrice': 19.95
+                }],
+                amount=19.95)
```
Comment (estj92) [inline docs/guide/transactions.rst]: What if this amount is different from the sum of orderLines.unitPrice?

Comment (Piidgey) [reply docs/guide/transactions.rst]: The orderlines are not used for the actual refund.. But some payment methods (Klarna) may use them.

**Pull request merged**